lib/quda_cuda_api.cpp

```diff
@@ -189,6 +189,9 @@ namespace quda {
       case cudaMemcpyDeviceToDevice:
         PROFILE(cuMemcpyDtoDAsync((CUdeviceptr)dst, (CUdeviceptr)src, count, stream), QUDA_PROFILE_MEMCPY_D2D_ASYNC);
         break;
+      case cudaMemcpyDefault:
+        PROFILE(cuMemcpyAsync((CUdeviceptr)dst, (CUdeviceptr)src, count, stream), QUDA_PROFILE_MEMCPY_D2D_ASYNC);
```

Comment:
Is `QUDA_PROFILE_MEMCPY_D2D_ASYNC` appropriate for default? I thought default was supposed to deduce the correct type of copy?